provisioner/kuberpak/internal/helm-operator-plugins/predicate/depedent.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2020 The Operator-SDK Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package predicate
+
+import (
+	"reflect"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	crtpredicate "sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+var log = logf.Log.WithName("predicate")
+
+type GenerationChangedPredicate = crtpredicate.GenerationChangedPredicate
+
+// DependentPredicateFuncs returns functions defined for filtering events
+func DependentPredicateFuncs() crtpredicate.Funcs {
```

Comment:
This one is internal in the `helm-operator-plugins` repo, so I made a copy of it in kuberpak. It might make sense to check if we can make that exported _somewhere_ so it can be shared.